### PR TITLE
Remove some options

### PR DESCRIPTION
**Pull request by fxcoudert**
Fewer builds from source, less CO₂ emitted, more users happy

- Some formulas with low install count and never-used options: writerperfect, inspircd, moc
- Formulas with often used options that carry no extra dependency (mpv) or few deps (sox)

### CODE REVIEW

File: Formula/mpv.rb [hunk 1]
```diff
@@ -84,10 +82,9 @@ def install
     system "python3", "waf", "configure", *args
     system "python3", "waf", "install"
 
-    if build.with? "bundle"
-      system "python3", "TOOLS/osxbundle.py", "build/mpv"
-      prefix.install "build/mpv.app"
-    end
+    # Build the app bundle
+    system "python3", "TOOLS/osxbundle.py", "build/mpv"
+    prefix.install "build/mpv.app"
```
Comment (ilovezfs) [inline Formula/mpv.rb]: Current policy is not to build .app GUI bits by default if they can be optional

Comment (fxcoudert) [reply Formula/mpv.rb]: Here it's used by 40% of users, triggering build from source that can be easily avoided (and at very low cost to other users)

Comment (ilovezfs) [reply Formula/mpv.rb]: Nonetheless, the eventual fate of GUI options is likely removal not default inclusion.

Comment (fxcoudert) [reply Formula/mpv.rb]: Just to be sure there is no misunderstanding: mpv builds the GUI anyway. It merely wraps it in a `.app` when using the option.

Comment (ilovezfs) [reply Formula/mpv.rb]: https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Formulae.md#stuff-that-builds-an-app

Comment (fxcoudert) [reply Formula/mpv.rb]: _“that can be distributed by Homebrew Cask”_: it won't be, because it builds from source and Cask does not like that

Comment (ilovezfs) [reply Formula/mpv.rb]: mpv is already in cask.

**Comment by fxcoudert** [comment]
`alure` test failure is preexisting

**Pull request closed without merging**